cdap-app-templates/cdap-etl/cdap-etl-api-spark/src/main/java/co/cask/cdap/etl/api/streaming/StreamingContext.java

```diff
@@ -37,4 +39,9 @@
    * @return CDAP JavaSparkExecutionContext for the pipeline.
    */
   JavaSparkExecutionContext getSparkExecutionContext();
+
+  /**
+   * Register lineage using the given reference name.
```

Comment:
javadoc should be expanded a little more, especially on the exceptions it can throw and what they mean.